packages/gql/codegen.ts

```diff
@@ -70,7 +70,7 @@ export const SUBGRAPHS = {
       return {
         //TODO: deploy subgraph on mainnet, polygon and arbitrum
         // https://linear.app/bleu-llc/issue/BAL-290/deploy-subgraph-with-token-relation-on-other-networks
-        [Network.Mainnet]: `${baseEndpoint}/balancer-v2-goerli`,
+        [Network.Mainnet]: `https://api.thegraph.com/subgraphs/name/bleu-llc/balancer-goerli-v2`,
```

Comment:
Is this really supposed to be Goerli?

Reply:
it's with the wrong name but it's deployed on the mainnet xD we have to redeploy with the correct name later, or that can be done automatically when Balancer accepts Fabio's changes on their repo